io.catenax.single_level_bom_as_specified/2.0.0/SingleLevelBomAsSpecified.ttl

```diff
@@ -0,0 +1,211 @@
+    samm:description "The description of the part in the primary language of the production facility of the product owner."@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "The steering wheel is nice and round".
+:partClassification a samm:Property;
+    samm:preferredName "Part Classification"@en;
+    samm:description "The part classification."@en;
+    samm:characteristic :PartClassificationCharacteristic.
+:lastModifiedOn a samm:Property;
+    samm:preferredName "Last Modified On"@en;
+    samm:description "The time the item was modified the last time"@en;
+    samm:characteristic samm-c:Timestamp;
+    samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+:createdOn a samm:Property;
+    samm:preferredName "Created On"@en;
+    samm:description "The time the item was created on"@en;
+    samm:characteristic samm-c:Timestamp;
+    samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+:partGeometry a samm:Property;
+    samm:preferredName "Part Geometry"@en;
+    samm:description "Some parts are intended to be used on either the left or right side, e.g. mirrors, hence this entity describes on which side a part is to be used"@en;
+    samm:characteristic :PartGeometryCharacteristic;
+    samm:exampleValue "left".
+:PartQuantityCharacteristic a samm:Characteristic;
+    samm:preferredName "Part Quantity Characteristic"@en;
+    samm:description "The characteristic of the part quantity."@en;
+    samm:dataType :PartQuantityEntity.
+:PartQuantityEntity a samm:Entity;
+    samm:preferredName "Part Quantity Entity"@en;
+    samm:description "The entity encapsulating the properties describing the quantity of a part."@en;
+    samm:properties (:measurementUnit :quantityNumber :measurementUnit :quantityNumber).
+:measurementUnit a samm:Property;
+    samm:preferredName "Measurement Unit"@en;
+    samm:description "Unit of measurement for the quantity of serialized objects."@en;
+    samm:characteristic samm-c:UnitReference;
+    samm:exampleValue "kW"^^samm:curie.
+:quantityNumber a samm:Property;
+    samm:preferredName "Quantity Number"@en;
+    samm:description "The number of objects related to the measurement unit."@en;
+    samm:characteristic :NumberOfObjects;
+    samm:exampleValue "350"^^xsd:double.
+:NumberOfObjects a samm:Characteristic;
+    samm:preferredName "Number of Objects"@en;
+    samm:description "Quantifiable number of objects in reference to the measurementUnit."@en;
+    samm:dataType xsd:double.
+:PartClassificationCharacteristic a samm-c:Set;
+    samm:preferredName "Part Classification Characteristic"@en;
+    samm:description "The characteristic of the part classification."@en;
+    samm:dataType :PartClassificationEntity.
+:PartClassificationEntity a samm:Entity;
+    samm:preferredName "Part Classification Entity"@en;
+    samm:description "The entity encapsulating the properties of the part classification."@en;
+    samm:properties (:key :value :key :value).
```

Comment:
why having key value twice? I assume a merge error and two of it can be removed.

Reply:
resolved